resources/views/admin/scholarship/partials/_form_scholarship.blade.php

```diff
@@ -157,8 +164,8 @@
 
 {{-- Recommendation Optional question --}}
 <div class="form-group">
-  {!! Form::label('display_optional_rec_question', 'Display optional recommendation question? ') !!}
-  {!! Form::checkbox('display_optional_rec_question', 1, ['class' => 'form-control']) !!}
+  {!! Form::label('display_optional_rec_question', 'Display optional recommendation question?') !!}
```

Comment:
Same same. Not vital, just seems funny to me as a question next to the checkbox.